### PR TITLE
Fix some links.

### LINKED ISSUE #1003: Link to the installation instructions on the Contributing page returns 404.
- Gitea version (or commit ref): fd941db246e66244ec81f43d74b8358c06173fd6
- Git version: 2.11.1
- Operating system: Arch Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ x] Yes (provide example URL)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description
Link to installation on the CONTRIBUTING.md page generates a 404.

## Screenshots
gitea-contributing
gitea-installation-404

(discussed on gitter, sending pull request).
**If regarding the Web Interface, please include a screenshot**

### PR DESCRIPTION
**Pull request by appleboy**
fix #1004 
fix #1003 

Also fix drone document link.

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-This document explains how to contribute changes to the Gitea project. It assumes you have followed the [installation instructions](https://github.com/go-gitea/docs/tree/master/en-US/installation). Sensitive security-related issues should be reported to [security@gitea.io](mailto:security@gitea.io).
+This document explains how to contribute changes to the Gitea project. It assumes you have followed the [installation instructions](https://docs.gitea.io/en-us/). Sensitive security-related issues should be reported to [security@gitea.io](mailto:security@gitea.io).
```
Comment (bkcsoft) [inline CONTRIBUTING.md:5]: This should point to the installation instructions, not just the docs-site

**Review by bkcsoft — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (lunny) [reply CONTRIBUTING.md:5]: Because installation don't has a land page...

**Comment by tboerger** [comment]
LGTM

**Pull request merged**